ghcide/src/Development/IDE/Plugin/CodeAction/ExactPrint.hs

```diff
@@ -211,17 +213,25 @@ extendImportTopLevel df idnetifier (L l it@ImportDecl {..})
     src <- uniqueSrcSpanT
     top <- uniqueSrcSpanT
     rdr <- liftParseAST df idnetifier
+
+    let alreadyImported =
+            showNameWithoutUniques rdr `elem`
+            map (showNameWithoutUniques @RdrName) (listify (const True) lies)
```

Comment:
I'm not sure `showNameWithoutUniques @RdrName` here works as expected, as it doesn't strip qualifier at least tested with Eval Plugin:

```hs
>>> showNameWithoutUniques (Qual (mkModuleName "Foo") $ mkVarOcc "++")
"Foo.++"
```

Perhaps it might work, as all `RdrName` occurring in `LIE`s could be `Unqual`, but I don't have confidense.
How about rewriting as `map (showNameWithoutUniques @OccName) (listify (const True) lies)`?

Reply:
Thanks, that's a good point. All `RdrName` in the `LIE` will be unqualified, that's guaranteed. 
However, `rdr` might be qualified, and if `showNameWithoutUniques` doesn't strip qualifiers, we'd have a bug.
I'll fix